src/components/Modal/LoginModal.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+function LoginModal() {
+  const { Kakao } = window;
+  const history = useHistory();
+
+  const handleKakaoLogin = () => {
+    Kakao.Auth.login({
+      success: function (response) {
+        fetch(`http://10.58.7.192:8000/users/kakaologin`, {
+          headers: {
+            Authorization: localStorage.getItem(
+              'kakao_a1d1dea38c0f71aaa5336c31c326efb0'
+            ),
+          },
+          method: 'POST',
+          body: JSON.stringify({ access_token: response.access_token }),
+        })
+          .then(res => res.json())
+          .then(res => {
+            localStorage.setItem('kakao_token', res.access_token);
+            if (res.access_token) {
+              alert('카카오 로그인 성공');
+              history.push('/');
+            }
+          });
+      },
+      fail: function (err) {
+        alert(JSON.stringify(err));
+      },
+    });
+  };
+
+  return (
+    <section>
+      <Container>
+        <Header>
+          <h1>wanted</h1>
+          <button>
+            <i className="far fa-window-close"></i>
+          </button>
+        </Header>
+        <Body>
+          <Intro>
+            <h1>
+              직장인을 위한 <br /> 커리어 플랫폼, 원티드!
+            </h1>
+            <h2>
+              커리어 성장과 행복을 위한 여정 <br /> 지금 원티드에서 시작하세요
+            </h2>
+          </Intro>
+          <Btnbox>
+            <button onClick={handleKakaoLogin}>
+              <img src="images/kakao_login_large_wide.png" alt="kakaoBtn" />
+            </button>
+            <button>Google로 시작하기</button>
+          </Btnbox>
+        </Body>
+      </Container>
+    </section>
+  );
+}
+
+const Container = styled.div`
+  position: relative;
+  max-width: 400px;
+`;
+
+const Header = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 16px 20 16px 20;
+  margin: 20px 0 0 20px;
+
+  h1 {
+    font-size: 26px;
+  }
+  button {
+    background: none;
+    border: 0;
+
+    i {
+      position: absolute;
+      font-size: 24px;
+      top: 0;
+      right: 10px;
+    }
+  }
+`;
+
+const Body = styled.div`
+  padding: 20px;
+`;
+
+const Intro = styled.div`
+  width: 360px;
+  height: 144px;
+  margin: 24px 0 40px 0;
+  text-align: center;
+
+  h1 {
+    line-height: 1.54;
+    font-size: 26px;
+    font-weight: 600;
+  }
+
+  h2 {
+    margin-top: 16px;
+    line-height: 1.5;
+    font-size: 16px;
+    font-weight: 400;
+    color: #666;
+  }
+`;
+
+const Btnbox = styled.div`
+  button {
+    background-color: transparent;
```

Comment:
```suggestion
const Button = styled.button`
  background-color: transparent;
```
BtnBox를 만들게 아니라 직접 Button을 만들면 될 것 같습니다